src/vue/forms/CreateFundForm.vue

```diff
@@ -359,6 +359,7 @@ const STEPS = {
 }
 const EVENTS = {
   close: 'close',
+  update: 'update',
```

Comment:
how about `submitted`?

Reply:
Actually, the form doesn't always emit that event after submitting. It does only when pending or rejected fund creation request is updated. So, maybe `updated`?

Reply:
Also, IMO it's better to avoid `CANCEL` button anywhere if it does not relate to the canceling a request. 

In this case, after clicking the cancel button the form emits `close` event while the same button on the `FundRequestDetails` page emits `cancel` event, that is related to the `cancelRequest` method.

I think it's bad UX, so we need either to remove the button and the event emitting from `CreateFundForm` or to redesign request cancellation button to avoid ambiguity.

Besides, since we need `close` event only for closing the form drawer, but the drawer has its own button for closing, the form event is useless.

Reply:
Updated, used `@request-updated` event